css/main.css

```diff
@@ -0,0 +1,165 @@
+.article-box div { margin-bottom: 0.75rem; }
+.article-box div:last-child { margin-bottom: 0; }
+.article-box a { font-size: 1.25rem; }
+
+/* Slideshow Section */
+.second-N { position: absolute; bottom: 6px; right: 14px; width: 36px; height: 36px; }
+
+.slide-caption {
+  position: absolute;
+  width: 68px;
+  height: 52px;
+  background-color: #fbfbfb;
+  bottom: 0;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: bold;
+  font-size: 1.25rem;
+  pointer-events: none;
+}
+.slide-caption-box { position: absolute; bottom: 0; right: 8px; width: 42px; height: 36px; border: 2px #df0000 solid; }
+.slide-caption-box-2 { position: absolute; bottom: 5px; right: 13px; width: 42px; height: 36px; border: 2px #df0000 solid; border-bottom: 0; border-right: 0; }
+.slide-caption-box-3 { position: absolute; bottom: 10px; right: 18px; width: 42px; height: 36px; border: 2px #df0000 solid; border-bottom: 0; border-right: 0; }
+
+/* Footer Subscribe and Newsletter Section */
+.book-case { padding: 122px 150px; position: relative; }
+.book-shelf { position: absolute; margin-bottom: 1rem; top: 0; left: 0; }
+.book-behind { position: absolute; z-index: 1; }
+.book-forefront { position: absolute; left: 90px; top: 13px; z-index: 2; }
+
+.subscribe-bg { background: url(../images/Footer/subscribe-img2.gif) center center / cover no-repeat; width: 48%; min-height: 160px; }
+
+.social-media { font-size: 28px; }
+.social-media a { margin-left: 0.65rem; color: gray; }
+.social-media a:first-of-type { margin-left: 0; }
+
+/* facebook icon */
+.social-media a:nth-child(1):hover { color: midnightblue; }
+
+/* twitter icon */
+.social-media a:nth-child(2):hover { color: skyblue; }
+
+/* instagram icon */
+.social-media a:nth-child(3):hover { color: blue; }
+
+/* tumbler icon */
+.social-media a:nth-child(4):hover { color: darkblue; }
+
+/* linkedIn icon */
+.social-media a:nth-child(5):hover { color: dodgerblue; }
+
+.footer-links { list-style: none; font-size: 0.9rem; font-weight: 500; color: #727272; }
+.footer-links li { border-right: 1px gray solid; padding: 0 0.85rem; line-height: 1rem; margin: 0.35rem 0; }
+.footer-links li:last-of-type { border: none; }
+.footer-links li a { color: #727272; }
+.footer-links li a:hover { color: #111; }
+
+/* Horizontal Line Dividers */
+.gray-divider { border-bottom: 2px #f2f2f2 solid; width: 96%; height: 2px; margin: auto; }
+
+/* Borders */
+.border-white { border: 1px #fff solid; }
+.border-red { border: 2px #df0000 solid; }
+
+/* Colors */
+.bg-white { background-color: #fff; }
+.bg-lgrey { background-color: #f2f2f2; }
+.bg-lgray { background-color: #f2f2f2; }
+.bg-grey-hover:hover { background-color: #d2d2d2; }
+
+/* Media queries */
+
+/* Small devices (portrait tablets and large phones, 600px and up) */
+@media screen and (max-width: 600px), (min-width: 600px) {
+  .d-show-sm-only { display: block; }
+  .d-show-md-only { display: none; }
+  .d-show-lg-only { display: none; }
```

Comment:
Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class name can be used instead. See [Display](https://getbootstrap.com/docs/4.1/utilities/display/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush: